reviews/review-priorities.md

```diff
@@ -16,12 +16,16 @@ Classifications are done through labels, e.g.: priority-high.
 
 Please note that these are the priorities used by the core team. External contributors are welcome to review PRs according to their own priorities, and such review activity will help highlight PRs which should get attention from the core team.
 
+## priority-very-high
+
+Same criteria as priority-high, below, but needed for the Mbed TLS teams' most important commitments.
```

Comment:
I feel that a forward reference is not very reader friendly. Also, as far as I understand the difference between the two is that `priority-high` is only a goal, whereas `priority-very-high` is a commitment. How about something like this:
> This classification is used for PRs which have a committed deadline for merging.

Reply:
`priority-very-high` is for diamond commitments (I haven't used that phrase here as it's meaningless outside of Arm / OSS), whereas `priority-high` is for standard commitments. Both have a deadline; the difference is that if a diamond is at risk, we might choose to take resource away from a non-diamond commitment to help.

But from an external point of view, they are pretty similar.

Reply:
Agree the forward reference isn't ideal, but it's tricky to avoid it without restructuring the whole thing.